doc/config/posix

```diff
@@ -0,0 +1,31 @@
+# Include common configuration options.
+@INCLUDE_PATH = doc/config
+@INCLUDE = common
+
+# Basic project information.
+PROJECT_NAME = "POSIX"
+
+# Library documentation output directory.
+HTML_OUTPUT = POSIX
+
+# Generate Doxygen tag file for this library.
+GENERATE_TAGFILE = doc/tag/POSIX.tag
+
+# Directories containing library source code.
+INPUT = doc \
+        doc/lib \
+        libraries/freertos_plus/standard/freertos_plus_posix/include \
+        libraries/freertos_plus/standard/freertos_plus_posix/source \
+		libraries/abstractions/posix/include/FreeRTOS_POSIX/sys \
+		vendors/pc/boards/windows/ports/posix \
```

Comment:
I doubt about port specific configs. It's nice to show some example, but may seem unnecessary...

Reply:
I agree. Removed!